building/tracks/practice-exercises.md

```diff
@@ -244,7 +244,7 @@ This exercise does not contain any error handling tests.
 
 This file contains meta information on the exercise:
 
-- `authors`: The exercise's author(s) (required)
```

Comment:
What about making them optional for historic reasons but required for any newly added practice exercise?

I don't know if that's easy to check with configlet, but for the purposes of building a track, i.e. adding new (practice) exercises, they should probably still be required?

Reply:
> I don't know if that's easy to check with configlet

I'd have to think this through.

Reply:
We'll postpone a decision on how to enforce authors for later.